### ISSUE TITLE
Upgrade dotnet sample EventHubsForKafkaSample to use dotnet 6

### ISSUE THREAD

**Issue by ViniciusSouza** [Azure/azure-event-hubs-for-kafka#198]
Description
===========
The sample code is still using .NET Core 2.0 it would be nice to have it updated to use the current version 

Checklist
=========

**IMPORTANT**: We will close issues where the checklist has not been completed or where adequate information has not been provided.

Please provide the relevant information for the following items:
 - [x] SDK (include version info): `Confluent.Kafka 1.8.2`
 - [x] Sample you're having trouble with: `Quick start dotnet EventHubsForKafkaSample`
 - [x] Namespace and desquad/quickstart
 - [x] Timestamps *in UTC* `June 3 2022 - 22:18:00 UTC`
 - [x] group.id or client.id `$Default`
 - [x] Operating system: `Windows 11 WSL 2 - Ubuntu 20.04`

If this is a question on basic functionality, please verify the following:
 - [x] Port 9093 should not be blocked by firewall ("broker cannot be found" errors)
 - [x] Pinging FQDN should return cluster DNS resolution (e.g. `$ ping namespace.servicebus.windows.net` returns ~ `ns-eh2-prod-am3-516.cloudapp.net [13.69.64.0]`)
 - [x] Namespace should be either Standard or Dedicated tier, not Basic (TopicAuthorization errors)
